### ISSUE TITLE
Final literals

### ISSUE THREAD

**Issue by srittau** [PyCQA/flake8-pyi#468]
We should warn about final literal constants that use `Final[Literal[...]]` instead of `: Final = ...`.

**Comment by tomasr8** [comment]
How would this work if you have multiple values e.g. `Literal[1, 2]`? Or is this just intended for single values?

**Comment by srittau** [comment]
@tomasr8 The latter. A final allowing multiple literal values wouldn't make much sense. To clarify, this is for cases, where the implementation has constants like this:

```python
FOO = 42
```

This should become `FOO: Final = 42` in the stubs, not `FOO: Final[Literal[42]]`.

**Comment by tomasr8** [comment]
I figured, just wanted to make sure I wasn't missing something. Thanks!
If you weren't planning to implement it yourself, I'd like to give it a try :)
